ui/app/AppLayouts/Wallet/popups/simpleSend/SendSignModal.qml

```diff
@@ -0,0 +1,308 @@
+        secondaryText: root.tokenSymbol !== Constants.ethToken ?
+                           SQUtils.Utils.elideAndFormatWalletAddress(root.tokenContractAddress) : ""
+        icon: Constants.tokenIcon(root.tokenSymbol)
+        badge: root.networkIconPath
+        highlighted: contractInfoButtonWithMenu.hovered
+        components: [
+            ContractInfoButtonWithMenu {
+                id: contractInfoButtonWithMenu
+
+                objectName: "contractInfoButtonWithMenu"
+                visible: root.tokenSymbol !== Constants.ethToken
+                symbol: root.tokenSymbol
+                contractAddress: root.tokenContractAddress
+                networkName: root.networkName
+                networkShortName: root.networkShortName
+                networkBlockExplorerUrl: root.networkBlockExplorerUrl
+                onOpenLink: (link) => root.openLinkWithConfirmation(link)
+            }
+        ]
+        visible: !root.isCollectible
+    }
+
+    // Send Collectible
+    ColumnLayout {
+        objectName: "sendCollectibleBox"
+        StatusBaseText {
+            objectName: "collectibleCaption"
+            text: qsTr("Send")
+            font.pixelSize: Theme.additionalTextSize
+        }
+        SignCollectibleInfoBox {
+            Layout.fillWidth: true
+            Layout.bottomMargin: Theme.bigPadding
+            name: !!root.collectibleName ? root.collectibleName: qsTr("Unknown")
+            backgroundColor: root.collectibleBackgroundColor
+            isMetadataValid: root.collectibleIsMetadataValid
+            fallbackImageUrl: root.collectibleFallbackImageUrl
+            contractAddress: root.collectibleContractAddress
+            tokenId: root.collectibleTokenId
+            networkShortName: root.networkShortName
+            networkBlockExplorerUrl: root.networkBlockExplorerUrl
+            loading: root.isCollectibleLoading
+            openSeaExplorerUrl: root.fnGetOpenSeaExplorerUrl(root.networkShortName)
+            onOpenLink: (link) => root.openLinkWithConfirmation(link)
+        }
+        visible: root.isCollectible
+    }
+
+    // From
+    SignInfoBox {
+        Layout.fillWidth: true
+        Layout.bottomMargin: Theme.bigPadding
+        objectName: "accountBox"
+        caption: qsTr("From")
+        primaryText: root.accountName
+        secondaryText: SQUtils.Utils.elideAndFormatWalletAddress(root.accountAddress)
+        asset.name: "filled-account"
+        asset.emoji: root.accountEmoji
+        asset.color: root.accountColor
+        asset.isLetterIdenticon: !!root.accountEmoji
+    }
+
+    /** TODO: Use new recipients appraoch from
+    https://github.com/status-im/status-desktop/issues/16916 **/
+    // To
+    SignInfoBox {
+        Layout.fillWidth: true
+        Layout.bottomMargin: Theme.bigPadding
+        objectName: "recipientBox"
+        caption: qsTr("To")
+        primaryText: root.recipientAddress
```

Comment:
@Khushboo-dev-cpp we should not display the recipient address always, we need some conditions. Check the Figma, but should be... display the account name if exists, if not display the saved address name if exists, if not display ens name if exists, if not display the address.

Reply:
@saledjenic Ive mentioned a todo
```
    /** TODO: Use new recipients appraoch from
    https://github.com/status-im/status-desktop/issues/16916 **/
```
    
    was waiting for the recipient part and mechanism to be finalised